packages/react-devtools-shared/src/devtools/views/Profiler/SidebarSelectedFiberInfo.js

```diff
@@ -75,7 +112,7 @@ export default function SidebarSelectedFiberInfo(_: Props) {
           <ButtonIcon type="close" />
         </Button>
       </div>
-      <div className={styles.Content}>
+      <div className={styles.Content} onKeyDown={handleKeyDown}>
```

Comment:
Looks like this event isn't firing for Firefox. I think that's because of a missing `tabIndex={0}`